### PR TITLE
Add a note about setting var. when using antigen

### PR DESCRIPTION
**Pull request by redianthus**
Hi,

Something like:

```sh
. /usr/share/zsh-antigen/antigen.zsh

GEOMETRY_SYMBOL_PROMPT="λ"

antigen use oh-my-zsh
antigen theme frmendes/geometry
```

won't work. But:

```sh
GEOMETRY_SYMBOL_PROMPT="λ"

. /usr/share/zsh-antigen/antigen.zsh

antigen use oh-my-zsh
antigen theme frmendes/geometry
```

is ok. So, I just added a note about it.

### CODE REVIEW

**Comment by frm** [comment]
Hi!

Thanks for this. Before merging, are you sure this just applies to antigen? Wouldn't it be similar in `zplug` (iirc)?

**Comment by redianthus** [comment]
I don't use/know zplug, so can't help here, sorry !

**Comment by desyncr** [comment]
It may be too specific to the plugin manager but I don't see too much of a trouble with adding it. 👍

**Comment by frm** [comment]
Oh, yeah, I'm up for merging! I was just wondering if we should change it to be broader, without even specifying a package manager. As an example: "(...) you should set them before loading the theme in your package manager (e.g. antigen)"

**Comment by desyncr** [comment]
Agree on that point.

**Comment by redianthus** [comment]
The fact is that it's not just about loading the theme in the package manager. In my first example, I kinda set them before loading the theme with the package manager but it doesn't work.

**Comment by desyncr** [comment]
@zapashcanon Can you clarify what works and what doesn't work?

I'm up for merging but text needs to be reworked in a generic way (example above is OK).

**Comment by redianthus** [comment]
It's very clear in my first post, no ?

You must set env. var. before loading antigen. Loading them before `antigen theme frmendes/geometry` is not enough.

**Comment by desyncr** [comment]
@zapashcanon What wasn't clear was your previous comment. Otherwise I wouldn't ask.

**Comment by redianthus** [comment]
Hum ok. It was just another way to explain what's in the first post.

**Comment by frm** [comment]
Any updates on this?

**Review by desyncr — CHANGES_REQUESTED** [review]
Re word in a non plugin specific way: "(...) you should set them before loading the theme in your package manager (e.g. antigen)"

**Comment by redianthus** [comment]
It's not really about setting them before loading the theme. (This is already in the `README`).

It's about setting them before "loading" the package manager. Sourcing `antigen.zsh` for antigen e.g., but I don't know how other package manager are loaded/work, so it may not be the case.

**Review by frm — APPROVED** [review]
We can merge this, I think. It's really a matter of semantics and if the issue arises in other package managers, we can update it.

Sorry for taking so long to reply @zapashcanon. Tough couple of months.

**Comment by desyncr** [comment]
@zapashcanon This seems to be an issue with this particular package manager rather than with geometry itself.

We think geometry should be agnostic to a package manager. It adds various install methods in its `README.md` but just as a convenience.

**Comment by frm** [comment]
@desyncr yeah, but if it affects users we might as well add in that information. It's not a big deal and it can avoid issues in the future for anyone who tries to install it.

Although I just noticed it's kinda lost in the middle of the `README`. I'd rephrase it and put it in the `antigen` specific section. What do you think?

**Comment by frm** [comment]
@zapashcanon can you take a look at my last comment?

**Comment by desyncr** [comment]
@fribmendes @zapashcanon The issue with Antigen is resolved so this gotcha is not necessary anymore.

**Comment by frm** [comment]
Should we close this then? @desyncr

**Comment by desyncr** [comment]
Yes. Thanks @zapashcanon for the work. It's currently solved in Antigen itself.
